workshops/rock_paper_scissors/README.md

```diff
@@ -0,0 +1,294 @@
+---
+Name: 'Rock Paper Scissors Game'
+Description: 'Simple rock paper scissors game using Python'
+Author: '@JackTDC'
+---
+
+# Rock Paper Scissors Game
```

Comment:
Remove this header, the title from “Name” will appear on the page header really nicely